### PR TITLE
new sub command: ssm cmd

### PR DESCRIPTION
**Pull request by giuliocalzolari**
ssm cmd YOURINSTANCENAME id

can be used to quickly launch multiple command to EC2 fleet

### CODE REVIEW

**Comment by giuliocalzolari** [comment]
@boroivanov

File: ssmrun/cli.py [hunk 1]
```diff
@@ -14,6 +14,60 @@
...
+@click.option('-k', '--target-key', default='Name', help='Target tag key (default: Name)')
+@click.option('-c', '--comment', default='ssmrun cli', help='Command invocation comment')
+@click.option('-i', '--interval', default=1.0, help='Check interval (default: 1.0s)')
+@click.option('-p', '--profile', default=None, help='AWS profile')
+@click.option('-r', '--region', default=None, help='AWS region')
+def cmd(target, command, show_stats, show_output, target_asg, target_stack, target_key, comment, interval, profile, region):
+    """Send SSM AWS-RunShellScript to target, quick emulation of virtual SSH interface"""
+    # Parse parameters for the SSM Command
+    ssm_document = "AWS-RunShellScript"
+    ssm_params = {"commands": command}
```
Comment (boroivanov) [inline ssmrun/cli.py]: I think `command` needs to be a list or tuple. I'm getting
```
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter Parameters.commands, value: uptime, type: <type 'unicode'>, valid types: <type 'list'>, <type 'tuple'>
```
After making the change bellow it works just fine.
```
-    ssm_params = {"commands": command}
+    ssm_params = {"commands": [command]}
```

**Comment by giuliocalzolari** [comment]
done

File: ssmrun/cli.py (hunk 1 above)
Comment (giuliocalzolari) [reply ssmrun/cli.py]: done

**Comment by boroivanov** [comment]
@giuliocalzolari ssmrun 0.1.4 is out. Thanks a lot for your help.

**Comment by giuliocalzolari** [comment]
Great thanks!
And btw I will do some others pr and also I would like to use library into
a Aws public speech to present a NOSSH approach!
Now with private endpoint the scenario are unlimited and can be used to
have PCI compliance!

**Comment by boroivanov** [comment]
Sounds good :). Let us know when you have speech. Our team is very interested to see what you have to present.

**Pull request merged**